### PR TITLE
Feaature/hdf5 intel

### PR DESCRIPTION
**Pull request by gjbex**
HDF5 for serial and parallel I/O, the latter with support for GPFS, intel toolchain 2015a

### CODE REVIEW

File: easybuild/easyconfigs/h/HDF5/HDF5-1.8.14-intel-2015a-gpfs.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+version = '1.8.14'
+versionsuffix = '-gpfs'
+
+homepage = 'http://www.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of 
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'pic': True, 'usempi': True}
+configopts = "--enable-gpfs"
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/HDF5/HDF5-1.8.14-intel-2015a-gpfs.eb]: move this next to `buildopts`

**Comment by wpoely86** [comment]
ok to test

**Comment by wpoely86** [comment]
@gjbex a typo in the filename

**Comment by gjbex** [comment]
Both issues fixed.  Maybe we can leave out the buildopts in the EB for the serial version since this only applies to MPI?

**Comment by wpoely86** [comment]
@gjbex you can probably drop the `-DMPICH_IGNORE_CXX_SEEK` in the serial version but it should be test to be sure :wink:

**Comment by gjbex** [comment]
Removed buildopts from serial version, tested, installs cleanly.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b7e5728c6607e901e911 for a full test report.

**Comment by boegel** [comment]
Looks great, testing works out OK, so good to go in.

Thanks for the review @wpoely86 and @gjbex for resolving the remarks!

**Pull request merged**